schema/aggregations/aggregation_count.js

```diff
@@ -19,7 +22,7 @@ export const AggregationCountType = new GraphQLObjectType({
   },
 });
 
-export default {
+export default ({
   type: AggregationCountType,
   resolve: ({ name, count }, id) => ({ id, name, count }),
-};
+}: GraphQLFieldConfig<AggregationCountType, any>);
```

Comment:
as we don't have an object, we use casting to set it e.g. ` ({}: Type)` ( [some examples](https://flowtype.org/docs/builtins.html#string) )